### ISSUE TITLE
Implement team viability measure

### ISSUE THREAD

**Issue by JamesPHoughton** [Watts-Lab/surveys#12]
One of the most important measures after a deliberation is the participant's subjective experience of working with the other participants. One survey that implements this is:

Cooperstein, Jessica Nicole. 2017. “Initial Development of a Team Viability Measure.” Edited by Suzanne T. Bell. Master of Arts, DePaul University. https://via.library.depaul.edu/cgi/viewcontent.cgi?article=1211&context=csh_etd.

There are a number of questions in this survey that all try to get at the same fundamental underlying concept, and we will collapse those measures into a single outcome variable. This may require us to write an aggregation function in the .json file that defines the survey, and modify the surveyWrapper to call this function before storing the data. See discussion here: https://github.com/Watts-Lab/surveys/issues/28

## Ready to close when
- [x] Implemented as a surveyJS survey
- [x] The survey is documented in associated mdx
- [x] Added as an exit step in empirica

**Comment by JamesPHoughton** [comment]
@markwhiting - when you have used this survey in the past, have you used all 20 questions in the paper, or have you used a subset?

**Comment by markwhiting** [comment]
I used the 14 question version in appendix E of the paper, which is also here → https://github.com/StanfordHCI/bang/blob/master/txt/midsurvey-q.txt

One of my students showed that we could collapse this into one question, but we didn't have enough data to validate so we have not published that.

**Comment by JamesPHoughton** [comment]
How did you go about figuring out which was the one question to ask? Not sure how one goes about doing this sort of thing. My ideal would probably be about 3-5 questions, I think, if we knew which ones.

Do you collapse the questions into a single measure, or use them as independent measures?

**Comment by markwhiting** [comment]
We collapsed into a single measure, as is standard with a Likert of this type. Let me see if I can get any helpful insight from the student who worked on the simplification bit.

**Comment by markwhiting** [comment]
My thoughtful student helped me realize that we'd actually included some initial analysis on this scale in a past paper → see pp 12–13 [here](https://dl.acm.org/doi/pdf/10.1145/3359311) for a few paragraphs on this. 

We started a validation process but it's not done yet. Let me know if someone here wants to take that on as a reasonably straightforward paper and I loop in the appropriate people.

**Comment by JamesPHoughton** [comment]
@sky1113 - lets implement the 14-item for our experiment this week. 

@markwhiting It would be great to do the full validation - let's outline it as a project for the fall.

**Comment by markwhiting** [comment]
Yep, sounds great! I'd say it's a rather good independent study project.

**Comment by JamesPHoughton** [comment]
Adding an issue for tasks for the fall, I expect this will become its own project eventually: https://github.com/Watts-Lab/deliberation-project/issues/38

**Comment by kepstein23** [comment]
Blocked on #28

**Comment by JamesPHoughton** [comment]
Lets split this into sub-issues:
- [ ] Write aggregation function for team viability survey
- [ ] Identify short version of team viability survey to use temporarily
